### ISSUE TITLE
Why should LLVM be built from source?

### ISSUE THREAD

**Issue by rafidka** [Alusus/Alusus#98]
I tried to build Alusus today and it started by building the whole of LLVM which took a very long time (more than an hour I believe) on my macOS (Mojave 10.14.6, 3.1 GHz Intel Core i7, 16 GB 1867 MHz DDR3). I don't see why it is necessary to build it when it is already built for multiple operating systems. Check for example the [release page for LLVM 7.0](https://github.com/llvm/llvm-project/releases/tag/llvmorg-7.1.0), the version which is currently used by Alusus. I suggest the build script is modified to support downloading the built image instead. Those who needs LLVM source code, i.e. Alusus main developers and contributors, can easily install it if they need.

**Comment by sarmadka** [comment]
You are right, it's probably not necessary to build LLVM from source. We just haven't had the time to spend on trying the pre-built version and updating the build script. Will keep it on the ToDo list. Contributions in this regard is welcome though :)
